File: python/exercício_35.py
print()
print("=-"*60)
print()
print("--------------Dias da semana------------------")
print()
print("=-"*60)
print() 
dia=int(input("Digite um valor de 1 a 7: "))

if dia == 1:
    print("domingo")
else:
    if dia == 2:
        print("Segunda-feira")
    if dia == 3:
        print("Terça-feira")
    if dia == 4:
        print("Quarta-feira")
    if dia == 5:
        print("Quinta-feira")
    if dia == 6:
        print("Sexta-feira")
    if dia == 7:
        print("Sabado")
if dia >7:
    print("Digite um valor valido")
if dia <1:
    print("Digite um valor valido")

File: java/exercício_35.java
import java.util.Scanner;

public class exercício_35 {
    public static void main(String[] args) {  
    
        int num;
    
      try (Scanner leitor = new Scanner(System.in)) {
        System.out.println("Digite um numero de 1 a 7: ");
          num = leitor.nextInt();
    }
      
      if(num > 7 | num < 1 ){
          
          System.out.println("O numero digitado nao corresponde ao solicitado ");
          System.exit(0);
        }
      
      if(num == 1){
      
          System.out.println("DOMINGO");
      }
      if(num == 2){
      
          System.out.println("SEGUNDA");
      }
      if(num == 3){
      
          System.out.println("TERCA");
      }
      if(num == 4){
      
          System.out.println("QUARTA");
      }
      if(num == 5){
      
          System.out.println("QUINTA");
      }
      if(num == 6){
      
          System.out.println("SEXTA");
      }
      if(num == 7){
      
          System.out.println("SABADO");
      }      
      
    }
}
